src/locale/locales/de/messages.po

```diff
@@ -873,19 +835,19 @@ msgstr "Fortfahren"
 #: src/screens/Onboarding/StepModeration/index.tsx:115
 #: src/screens/Onboarding/StepTopicalFeeds.tsx:105
 msgid "Continue to next step"
-msgstr ""
+msgstr "Weiter zum nächsten Schritt"
 
 #: src/screens/Onboarding/StepAlgoFeeds/index.tsx:167
 msgid "Continue to the next step"
-msgstr ""
+msgstr "Weiter zum nächsten Schritt"
 
 #: src/screens/Onboarding/StepSuggestedAccounts/index.tsx:191
 msgid "Continue to the next step without following any accounts"
-msgstr ""
+msgstr "Fahren Sie mit dem nächsten Schritt fort, ohne die Konten zu verfolgen"
```

Comment:
```suggestion
msgstr "Fahren Sie mit dem nächsten Schritt fort, ohne Konten zu folgen"
```

"verfolgen" would mean like "observe" instead of "follow". Also, there is no need for the preposition here.